test/integration.js

```diff
@@ -213,12 +213,57 @@ describe('hoodie.account', function () {
     }, newUsername).then(toValue)
     .should.eventually.have.property('username', newUsername)
 
-    // .executeAsync(function fetch (done) {
-    //   hoodie.account.fetch().then(done, done)
-    // }).then(toValue)
-    // .then(function (account) {
-    //   expect(account.username).to.equal('username')
-    // })
```

Comment:
this code did not belong in here, but I moved it into it’s own test: `.fetch() resolves with account properties`